src/module.ts

```diff
@@ -76,6 +76,7 @@ const defaultsByBackend: {
       endpoint: { path: '/refresh', method: 'post' },
       refreshOnlyToken: true,
       token: {
+        refreshResponseTokenPointer: '/token',
```

Comment:
So personally I think we should apply the following logic for setting this value to ensure it does not result in breaking changes:

- if `token.refreshResponseTokenPointer` is set, we use it to retrieve the token
- if `token.refreshResponseTokenPointer` is not set, we use `signInResponseRefreshTokenPointer` to retrieve the token instead of setting a default value

This is primarily to avoid introducing a breaking change, as if someone set `signInResponseRefreshTokenPointer` to e.g. `/accessToken` in their setup, and uses a refresh token, then their refresh response at the moment also returns the same schema as their sign in request.

Also cc @phoenix-ru on your opinion

Reply:
I totally agree with you. 

Only one thing: when the `token.refreshResponseTokenPointer` is not set, we have to use `signInResponseTokenPointer` and not `signInResponseRefreshTokenPointer`. 
(You probably meant the same thing but just wrote the wrong option 😊)

Reply:
Haha, yes! Slight slipup from my side. You are completely right!

Reply:
Obviously with this new implementation the default value for `refreshResponseTokenPointer` has to be empty, do you agree?

Reply:
> Obviously with this new implementation the default value for refreshResponseTokenPointer has to be empty, do you agree?

Yup! We have a default set for the `signInResponseTokenPointer` which would be used as a fallback!